src/test/java/com/commercetools/project/sync/producttype/ProductTypeSyncerTest.java

```diff
@@ -67,4 +85,35 @@ void getQuery_ShouldBuildProductTypeQuery() {
     // assertion
     assertThat(query).isEqualTo(ProductTypeReferenceResolutionUtils.buildProductTypeQuery(1));
   }
+
+  @Test
+  void syncWithError_ShouldCallErrorCallback() {
```

Comment:
Please make a better name here including `where` clause. Sth like `syncWithError_WhenNoKeyIsProvided_ShouldCallErrorCallback`. Also update it for other tests.

Reply:
created a new PR and please check this commit for the fix. https://github.com/commercetools/commercetools-project-sync/pull/213/commits/8030e3bdb0a9252deeb2e8a2b519333cc58d0861